lightning-block-sync/src/lib.rs

```diff
@@ -579,10 +576,8 @@ mod chain_notifier_tests {
 		let chain_listener = &MockChainListener::new()
 			.expect_block_connected(*chain.at_height(2))
 			.expect_block_connected(*new_tip);
-		let mut notifier = ChainNotifier {
-			header_cache: &mut chain.header_cache(0..=1),
-			chain_listener,
-		};
+		let mut notifier =
+			ChainNotifier { header_cache: &mut chain.header_cache(0..=1), chain_listener };
```

Comment:
Would be preferable if it deferred to the author for how to format a struct initializer, IMHO.